taxonomies/categories.txt

```diff
@@ -81554,7 +81571,7 @@ protected_name_type:en: pdo
 origins:en: en:Finland
 #wikidata:en:
 
-<en:Meat-based products
+<en:Meats and their products
```

Comment:
That one is (and was) probably incorrect.

Reply:
It's corrected, thank you